src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,25 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import strategy.MoveStrategy;
+
+public class CarTest {
+
+    @Test
+    void alwaysMove() {
+        MoveStrategy alwaysMoveStrategy = () -> true;
+        Car car = new Car(new CarName("gglee"), new Position(1), alwaysMoveStrategy);
+        car.move();
+        assertThat(car.getPosition()).isEqualTo(new Position(3));
```

Comment:
자동차가 1번 전진하면 위치 값은 2가 되어야 할 것 같습니다.
```suggestion
        assertThat(car.getPosition()).isEqualTo(new Position(2));
```